aiohttp_devtools/runserver/serve.py

```diff
@@ -65,16 +65,24 @@ async def on_prepare(request, response):
         @web.middleware
         async def static_middleware(request, handler):
             static_url = 'http://{}:{}/{}'.format(get_host(request), config.aux_port, static_path)
-            dft_logger.debug('settings app static_root_url to "%s"', static_url)
-            request.app['static_root_url'].change(static_url)
+            dft_logger.debug('setting app static_root_url to "%s"', static_url)
+            def set_static_url(app, url):
+                app['static_root_url'].change(url)
+                for subapp in app._subapps:
+                    set_static_url(subapp, url)
+            set_static_url(request.app, static_url)
```

Comment:
I'm not sure what the purpose of this middleware is (given that the value is already set below), but I assume it should do the same thing.